shared/wallets/banner/container.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import Banner from '.'
```

Comment:
I'm not sure `Banner` will really need a container, in `ConfirmScreen` I used it without and passed in the props when needed. Because `ConfirmScreen` uses the banner props to determine whether it should be rendered and because it's only one level of prop passing, I think this is fine. I left the container as to not break other files, though.